src/components/UserSearch.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+import {Route} from 'react-router-dom';
+
+class UserSearch extends Component {
+  constructor(props) {
```

Comment:
By based on the comment below, once the local state is global, the constructor can be removed.

Reply:
Okay sweet.